backend/.env.example

```
@@ -141,6 +141,8 @@ WEBHOOK_RETRY_COUNT=1
 WEBHOOK_RETRY_DELAY=1000
 # Max notifications per user per hour | integer | # Optional (default: 10)
 NOTIFICATION_RATE_LIMIT_PER_HOUR=10
+# Secret used to sign outbound webhook payloads and verify inbound callbacks | string (min 32 chars) | # Optional (default: empty)
+WEBHOOK_SIGNING_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEBHOOK_SIGNING_SECRET key should go before the WEBHOOK_TIMEOUT key

(UnorderedKey)